Packs/MicrosoftTeams/Integrations/MicrosoftTeams/MicrosoftTeams_test.py

```diff
@@ -3478,3 +3478,115 @@ def test_process_ask_user():
         },
     }
     assert process_ask_user(json.dumps(message)) == expected_adaptive_card
+
+
+def test_send_notification_with_raw_adaptive_card(mocker, requests_mock):
+    """
+    Given:
+        - A raw adaptive_card input.
+    When:
+        - send-notification is called.
+    Then:
+        - The adaptive card is wrapped with the required contentType and content fields.
+        - The request to the endpoint is sent with the adaptive card as expected.
+    """
+
+    from MicrosoftTeams import send_message
+
+    adaptive_card: dict = {
+        "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+        "version": "1.0",
+        "type": "AdaptiveCard",
+        "msteams": {"width": "Full"},
+        "body": [{"type": "TextBlock", "text": "message", "wrap": True}],
+        "actions": [
+            {"type": "Action.Submit", "title": "Yes"},
+            {"type": "Action.Submit", "title": "No"},
+        ],
+    }
+    expected_request_attachment = {
+        "contentType": "application/vnd.microsoft.card.adaptive",
+        "content": {
+            "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
+            "version": "1.0",
+            "type": "AdaptiveCard",
+            "msteams": {"width": "Full"},
+            "body": [{"type": "TextBlock", "text": "message", "wrap": True}],
+            "actions": [
+                {"type": "Action.Submit", "title": "Yes"},
+                {"type": "Action.Submit", "title": "No"},
+            ],
+        },
```

Comment:
can we use here adaptive_card? instead of the dict again ? I think it will be clearer.